valhalla/tyr/serializers.h

```diff
@@ -109,14 +109,18 @@ namespace osrm {
  */
 valhalla::baldr::json::MapPtr
 waypoint(const valhalla::Location& location, bool is_tracepoint = false, bool is_optimized = false);
+valhalla::baldr::json::MapPtr waypoint(const valhalla::Location& location,
```

Comment:
Can we add minimal documentation on what this does differently than the previous `waypoint` function (I know this takes `TripLeg_Edge` as an extra argument, but what does it do with it)?